DashAI/back/plugins/utils.py

```diff
@@ -102,38 +102,7 @@ def get_available_plugins() -> List[type]:
     return plugins_list
 
 
-def register_new_plugins(
-    component_registry: ComponentRegistry, available_plugins: List[type]
-) -> List[type]:
-    """
-    Register only new plugins in component registry
-
-    Parameters
-    ----------
-    component_registry : ComponentRegistry
-        The current app component registry
-
-    Returns
-    ----------
-    List[type]
-        A list of plugins' classes that were registered in the component registry
-    """
-    installed_plugins_set = {
-        component["class"] for component in component_registry.get_components_by_types()
-    }
-    available_plugins_set = set(available_plugins)
-    new_plugins = available_plugins_set - installed_plugins_set
-    for plugin in new_plugins:
-        # The component shouldnt be registered if it does not inherit from
-        # any DashAI base class with a 'TYPE' class attribute.
-        try:
-            component_registry.register_component(plugin)
-        except Exception as e:
-            logging.exception(e)
-    return list(new_plugins)
-
-
-def install_plugin_from_pypi(pypi_plugin_name: str) -> None:
+def install_plugin_from_pypi(pypi_plugin_name: str, install: bool = True) -> None:
     """
     Register only new plugins in component registry
 
```

Comment:
It wouldn't be better to call this method `call_pip` or `execute_pip_command` and set the install param as operation, which could take `install` or `uninstall` strings as values?
Also, remember to update the method docstring.

Reply:
Changed! The method name is execute_pip_command now, and the install or uninstall action depends of a string.